scene/gui/color_picker.cpp

```diff
@@ -186,9 +201,18 @@ void ColorPicker::_update_presets() {
 
 	preset->draw_texture_rect(get_icon("preset_bg", "ColorPicker"), Rect2(Point2(), preset_size), true);
 
-	for (int i = 0; i < presets.size(); i++) {
-		preset->draw_rect(Rect2(Point2(size.width * i, 0), size), presets[i]);
+#ifdef TOOLS_ENABLED
+	if (Engine::get_singleton()->is_editor_hint()) {
+		PoolColorArray arr_to_save = PoolColorArray();
+
+		for (int i = 0; i < presets.size(); i++) {
+			preset->draw_rect(Rect2(Point2(size.width * i, 0), size), presets[i]);
```

Comment:
Like this added presets will only show up in the editor. This needs to be outside the ifdef.

Reply:
Good catch, I need to pay more attention... Fixed